packages/payload-auth/src/better-auth/plugin/lib/build-collections/users/index.ts

```diff
@@ -71,6 +71,7 @@ export function buildUsersCollection({ incomingCollections, pluginOptions, schem
       admin: { description: 'Whether the email of the user has been verified' }
     }),
     name: () => ({
+      required: false,
```

Comment:
This comes required from better auth, so we should instead add the field instead and let this be required.